modules/auxiliary/admin/scada/yokogawa_bkbcopyd_client.rb

```diff
@@ -92,12 +92,6 @@ def send_pkt(data)
     return data
   end
 
-  def valid_response?(data)
```

Comment:
Given this removal does that mean `include Msf::Exploit::Remote::TcpServer` can be removed as well?

Reply:
This has nothing to do with any mixins. It's copypasta from the other `yokogawa_*` modules (which do call it).